Linkbus/Linkbus/RouteController.swift

```diff
@@ -40,7 +40,7 @@ extension RouteController {
         
         dispatchGroup.enter()
         fetchLinkbusApi { apiResponse in
-            if let success = apiResponse {
```

Comment:
Let's keep this in for now so that we can use success for network error catching